HAB5/hab5.tex

```diff
@@ -0,0 +1,404 @@
+EE skill set varies greatly based on method of implementation---purchasing ready-to-assemble products to design and fabrication of custom PCBs. 
+The result is that we have the opportunity to tune implementation based on the EEs we find. 
+Understanding of basic circuits and connections/interfacing is still required. 
+An understanding of the fundamentals of heat transfer is recommended in order to do basic thermal analysis. 
+The HAB structure leverages HAB4 designs so the brunt of mechanical design will focus on camera mounting and enclosures. 
+University Physics II (with optics) is recommended for imaging systems testing and calibration tasks.
+
+\subsection{Funding Requirements}
+% Payload:  ~$200 (hard cap at $300)
+% 2x raspberry pi 3 = $100 @ $40-50 each
+% 1x pi camera v2 = $28
+% 1x pi camera noir = $25
+% An arduino FPGA is a possible but unlikely inclusion, with a price tag of $60. The present budget also leaves room for the physical hardware of the payload including camera mounts and possible printing expenses. A filter setup, a la TetraPi, is deemed to be prohibitively expensive.
+
+% 	1x vis filter (reddish) $100 https://www.thorlabs.com/thorproduct.cfm?partnumber=FB650-40 
+% 	1x nir filter = $100 https://www.thorlabs.com/thorproduct.cfm?partnumber=FB880-70 
+
+This estimate is based off of the inclusion of a prior HAB board for baseline functionality, an existing APRS module and some functional WUAP-I hardware. 
+The cost of the balloon and helium are not included. 
+Significant leeway was left specifically because the exact implementation topology(for processing) and level of pre-fabrication of components scales inversely with availability of time dedicated by advanced electrical engineering students.
+
+\subsection{Faculty Support}
+The project laid forth doesn't assume that faculty support is guaranteed or that any particular faculty members will be forthcoming. 
+The project is undertaken in part hoping to build these connections without any significant one-way reliance. 
+It is for that reason that specific, informed and meaningful discussions with possible candidates are recommended as the project progresses and substantive results can be shared.
+
+\subsection{Long-term Vision}
+As laid out in the \autoref{primaryobjectives} and \autoref{benefittospex}, this document is tabled with an eye to the future. Data collection will focus not just on raw data but also on how processing develops and compares to the knowns. 
+Cataloguing the progress, as described in \autoref{deliverables}, builds on present resources and knowledge in a manner amenable to future growth. 
+SPEX's in-house ability is developed in line with estimates for skills required for later missions. 
+
+It is also held that missions be pursued on the basis of ability to draw in more permanent and sustainable sources of recognition and funding. 
```

Comment:
This should be moved to Benefit to SPEX.

Also work defined in the [HAB-CV PDD](https://github.com/RIT-Space-Exploration/SPEX-Project-Definition-Documents/blob/master/HAB-CV/hab-cv.pdf) should be referenced in this section.